docs/ref/contrib/gis/geoquerysets.txt

```diff
@@ -148,10 +148,15 @@ Backend     SQL Equivalent
 ==========  ============================
 PostGIS     ``ST_Contains(poly, geom)``
 Oracle      ``SDO_CONTAINS(poly, geom)``
-MySQL       ``MBRContains(poly, geom)``
+MySQL       ``ST_Contains(poly, geom)``
 SpatiaLite  ``Contains(poly, geom)``
 ==========  ============================
 
+.. versionchanged:: 3.0
+
+    MySQL now uses ``ST_Contains`` instead of ``MBRContains`` and operates on
```

Comment:
The usual wording is something like, "In older versions, MySQL uses MBRContains instead of ST_Contains and operates only on bounding boxes."